test/routes/entry-point.test.ts

```diff
@@ -0,0 +1,29 @@
+import jsonld from 'jsonld';
+import entryPoint from '../../src/routes/entry-point';
+import createContext from './context';
+
+describe('entry-point', (): void => {
+  it('should return the entry point', async (): Promise<void> => {
+    const context = createContext('entry-point');
+    const next = jest.fn();
+
+    await entryPoint(context.router)(context, next);
+
+    const { response } = context;
```

Comment:
this part is verbose, compared to the scope of executing a single request. Extract an abstraction?

Reply:
Attempted in https://github.com/libero/article-store/pull/10/commits/a86ac8f0f1423d6f0315af53cbcb45a9c8cc37cf.